database/database.go

```diff
@@ -2,14 +2,14 @@ package database
 
 import (
 	"log"
-	"os"
+
 	"github.com/InfluxCommunity/influxdb3-go/influxdb3"
 )
 
 func ConnectToDB() (*influxdb3.Client, error) {
-	url := os.Getenv("INFLUXDB_URL")
-	token := os.Getenv("INFLUXDB_TOKEN")
-	database := os.Getenv("INFLUXDB_DATABASE")
+	url := "https://us-east-1-1.aws.cloud2.influxdata.com"
```

Comment:
Please, do not change how env are implemented for security and operate in production in a differebt db of development.

Reply:
During the development we saw that we unfortunately have commited it... We revoked the key present, and returned to the .env specification in a future commit!

Reply:
*This PR is already with the updated commit with the .env structures